ja2003.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Jahresarbeit 2003
+Comment=Find die größte Fläche von zusammenhängenden Quadraten
```

Comment:
any chance to get these description in English too?
desktop files support localisation

Reply:
The program itself is mostly hardcoded in German. Does it make sense to have the description multi-lingual if the application only supports a single language?

Reply:
I think I wouldn't bother with the desktop file, as long as appdata is in English and mentions the application is only in German.